README.md

```diff
@@ -6,16 +6,14 @@ This repository is an overview for people who are getting involved in the projec
 
 ## Get Involved
 
-### Development
-
-If you'd like to help with ongoing development of these tools, great!
+If you'd like to join the community and help improve these tools, great!
 
 1. Review our [Contributor Guidelines](./CONTRIBUTING.md) and [Code of Conduct](./CONDUCT.md)
-1. Jump on our chat [archivers.slack.com](https://archivers.slack.com/), anyone can request an invite from [archivers-slack.herokuapp.com slackin](https://archivers-slack.herokuapp.com/)
-    - Contributor and development conversations happen on `#dev`
+1. Jump on the [Archivers chat (archivers.slack.com)](https://archivers.slack.com/), anyone can request an invite from [archivers-slack.herokuapp.com slackin](https://archivers-slack.herokuapp.com/)
+    - Introduce yourself in #Contributor and Development conversations happen on `#dev`
     - Ping one of the EDGI coordinators (@mattprice or @dcwalk) with your GitHub name to be added to the organization
 1. Take a look at our [Current Projects](#projects) and [Kanban Board](https://github.com/edgi-govdata-archiving/overview/projects/2)
-1. Join us at our [**weekly standup**](./protocol/standups.md), Saturdays at 6:30 ET (Eastern Time), call link posted in `#dev` as well as notes and recorded meetings
+1. Join us at our [**weekly community standup**](./protocol/standups.md), Thursdays at 6:30 ET (Eastern Time), call link posted on our [events calendar](https://envirodatagov.org/events/). If you've missed a call check out our [notes and recorded meetings](https://github.com/edgi-govdata-archiving/overview/blob/master/notes/_standup-summaries.md).
```

Comment:
"standup _meeting_" maybe? Not sure how clear it is what a "standup" is to non-tech people.